### DISCUSSION TITLE
Installation package for MacOS with intel chip

### DISCUSSION THREAD

**Discussion by Zeytuni** [RosettaCommons/rosetta#262] [PyRosetta]
Where can I find the installation package and instructions for PyRosetta on an Intel-based Mac? The download page and documentation only provide details for macOS with an M chip.

Thank you

**Comment by lyskov** [comment]
Please refer to https://www.pyrosetta.org/downloads for various ways to install PyRosetta for both Intel and Apple-silicon based Mac's.
